File: Anul 2/Medii de Programare si Proiectare/Lab/mpp-proiect-java-Mothyque/backend/Common/src/main/java/org/example/domain/ClientTicketDTO.java
package org.example.domain;

public class ClientTicketDTO extends Entity<Integer>
{
    private Client client;
    private Match match;
    private int numberOfTickets;

    public ClientTicketDTO(Client client, Match match, int numberOfTickets)
    {
        this.client = client;
        this.match = match;
        this.numberOfTickets = numberOfTickets;
    }

    public Client getClient()
    {
        return client;
    }

    public Match getMatch()
    {
        return match;
    }

    public int getNumberOfTickets()
    {
        return numberOfTickets;
    }

    public void setNumberOfTickets(int numberOfTickets)
    {
        this.numberOfTickets = numberOfTickets;
    }

    @Override
    public String toString()
    {
        return "Client: " + client.getName() + ", Address: " + client.getAddress() + ", Match: " + match.toString() + ", Number of tickets: " + numberOfTickets;
    }
}

File: Anul 2/Medii de Programare si Proiectare/Lab/mpp-proiect-csharp-Mothyque/ProiectMPP/ProiectMPP.Common/Domain/ClientTicketDTO.cs
using System;

namespace ProiectMPP.Common.Domain
{
    [Serializable]
    public class ClientTicketDTO : Entity<int>
    {
        public Client Client { get; set; }
        public Match Match { get; set; }
        public int NumberOfTickets { get; set; }

        public ClientTicketDTO() { }

        public ClientTicketDTO(Client client, Match match, int numberOfTickets)
        {
            Client = client;
            Match = match;
            NumberOfTickets = numberOfTickets;
        }

        public override string ToString()
        {
            return $"Client: {Client.Name}, Address: {Client.Address}, Match: {Match.ToString()}, Number of tickets: {NumberOfTickets}";
        }
    }
}
